lib/Driver.cpp

```diff
@@ -46,8 +48,10 @@ using namespace std::chrono;
 /// Each result contains result of execution of an original test and
 /// all the results of each mutant within corresponding MutationPoint
 
-std::vector<std::unique_ptr<TestResult>> Driver::Run() {
+std::unique_ptr<Result> Driver::Run() {
   std::vector<std::unique_ptr<TestResult>> Results;
+  std::vector<std::unique_ptr<Testee>> allTestees;
+  allTestees.reserve(100000);
```

Comment:
can be removed, copy and paste artefact for vector concatenation I didn't know how to do in CPP 😝